### PR TITLE
Removed Atom from Editors

### PR DESCRIPTION
**Pull request by martialblog**
- Atom will be discontinued https://github.blog/2022-06-08-sunsetting-atom/
 - Added Emacs (magit) in overview

### CODE REVIEW

**Review by mocdaniel — APPROVED** [review]

File: day2/02_Integrations/02_IDEs.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 !SLIDE smbullets
 # IDEs
 
-* Atom
 * JetBrains PHPStorm, ...
 * Visual Studio
+* VSCodium
```
Comment (theFeu) [inline day2/02_Integrations/02_IDEs.md:6]: I think we should use the full name Visual Studio Code - or am I thinking of something else right now?

Comment (mocdaniel) [reply day2/02_Integrations/02_IDEs.md:6]: afaik VSCodium is the OSS variant (MS-telemetry free).

Comment (martialblog) [reply day2/02_Integrations/02_IDEs.md:6]: Yeah I think you're correct

Comment (martialblog) [reply day2/02_Integrations/02_IDEs.md:6]: I will fix it on this branch

Comment (martialblog) [reply day2/02_Integrations/02_IDEs.md:6]: Done :heavy_check_mark:

**Comment by theFeu** [comment]
There is still an old screenshot in the training that needs to be switched out, I gave those to @htriem in #141

If someone has VSC installed, feel free to upload a new scrennshot of the integration (and the others) :)

**Comment by martialblog** [comment]
I updated the other PR. Let's not feature creep this one

**Pull request merged**